From 2f6de67a1831308fbb4fc883e5546fb712dde6f7 Mon Sep 17 00:00:00 2001
From: Michal Suchanek <msuchanek@suse.de>
Date: Thu, 22 Mar 2018 17:41:33 +0100
Subject: [PATCH] KVM: PPC: Book3S HV: Disable tb_offset

References: bsc#1086196
Patch-mainline: no, testing

This seems to improve timing sensitive workloads running in KVM guests.

Signed-off-by: Michal Suchanek <msuchanek@suse.de>
---
 arch/powerpc/kvm/book3s_hv.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/powerpc/kvm/book3s_hv.c b/arch/powerpc/kvm/book3s_hv.c
index 899b3a769082..aaa2045b6385 100644
--- a/arch/powerpc/kvm/book3s_hv.c
+++ b/arch/powerpc/kvm/book3s_hv.c
@@ -1642,6 +1642,7 @@ static int kvmppc_set_one_reg_hv(struct kvm_vcpu *vcpu, u64 id,
 		r = set_vpa(vcpu, &vcpu->arch.dtl, addr, len);
 		break;
 	case KVM_REG_PPC_TB_OFFSET:
+		break;
 		/*
 		 * POWER9 DD1 has an erratum where writing TBU40 causes
 		 * the timebase to lose ticks.  So we don't let the
-- 
2.13.6

